### DISCUSSION TITLE
right sidebar in docs

### DISCUSSION THREAD

**Discussion by ghost** [spacedriveapp/spacedrive#663] [Feature Request]
**what:**
right sidebar displaying page content among other things. widely used (t3, stripe, sveltekit etc...) and will be a nice feature as docs grow. 

**example:**

https://user-images.githubusercontent.com/98732287/229164798-850f64a6-c97b-4faf-9569-cf7d1f863a58.mov

**Comment by utkubakir** [comment]
Great idea!
